test-data/typecheck/test_serializers.yml

```diff
@@ -8,4 +8,15 @@
             class Meta:
                 pass
         class SerializerC(SerializerA, SerializerB):
-            pass
\ No newline at end of file
+            pass
+
+- case: model_serializer_meta_attributes
```

Comment:
Thanks a lot! This is a nice addition.

But, I was talking about some real serialiser definition that uses `ModelSerialiser` and `Meta.model` class.

Because I fear that `Meta` can use nominal subtyping and cause `mypy` to raise here.

Reply:
Okay, I've added another test. I'm not sure if it fits into test case you imagined.

I don't understand in which case `Meta` can use the nominal subtyping, can you elaborate on it?

Reply:
> Okay, I've added another test. I'm not sure if it fits into test case you imagined.
Oh yea, I get it because I used the `ModelSerializerMeta`, I've changed it to:
```python
class ModelSerializer(Serializer):
    class Meta:
        model: Type[Model]
        fields: Optional[Union[Sequence[str], str]]
        read_only_fields: Optional[Sequence[str]]
        exclude: Optional[Sequence[str]]
        depth: Optional[int]
        extra_kwargs: Dict[str, Dict[str, Any]]
    ...
```